### PR TITLE
feat(core): adds get method for QueryList

### PR DESCRIPTION
**Pull request by is2ei**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number:  #29467

## What is the new behavior?
By using `get` method, the developer can get the element by `index`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/core/src/linker/query_list.ts [hunk 1]
```diff
@@ -63,6 +63,13 @@ export class QueryList<T> implements Iterable<T> {
     if (!proto[symbol]) proto[symbol] = symbolIterator;
   }
 
+  /**
+   * Returns an item of the internal results.
```
Comment (IgorMinar) [inline packages/core/src/linker/query_list.ts:67]:
```suggestion
   * Returns the QueryList entry at `index`.
```

**Review by IgorMinar — DISMISSED** [review]
otherwise LGTM for public-api change

Reviewed-for: public-api

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: public-api
Reviewed-for: fw-core

**Comment by pkozlowski-opensource** [comment]
@is2ei thnx for this PR. We would like to move it forward, so could you please:
* address @IgorMinar feedback 
* amend the commit message so it references the issue (something like fixes #29467)

Thnx!

**Comment by mhevery** [comment]
[presubmit](http://test/OCL:342902585:BASE:342902608:1605640233116:87b7b7f1)

**Review by mhevery — APPROVED** [review]

File: packages/core/src/linker/query_list.ts (hunk 1 above)
Comment (mhevery) [reply packages/core/src/linker/query_list.ts]: change incorporated

**Review by mhevery — APPROVED** [review]
reviewed-for: global-approvers

**Pull request closed without merging**